lms/templates/courseware/static_tab.html

```diff
@@ -9,13 +9,13 @@
 <%block name="bodyclass">view-in-course view-statictab ${course.css_class or ''}</%block>
 <%namespace name='static' file='/static_content.html'/>
 
-<%block name="headextra">
+<%block name="head_extra">
 <%static:css group='style-course-vendor'/>
 <%static:css group='style-course'/>
 ${HTML(fragment.head_html())}
 </%block>
-<%block name="js_extra">
 
+<%block name="footer_extra">
```

Comment:
You have `footer_extra` here, and `foot_extra` below.  I'd choose one and go for it.